### DISCUSSION TITLE
在ShokaX中有哪些功能您很少使用

### DISCUSSION THREAD

**Discussion by zkz098** [theme-shoka-x/hexo-theme-shokaX#177] [Polls]
此投票调查功能使用率，以便确定渲染器重构中哪些功能为非核心功能，便于分包处理
此投票仅包含渲染器的特性

如果所有功能您均在积极使用，则建议您不投票。

**Comment by zkz098** [comment]
有两个及以上的可以发评论，github这个投票好像不支持多选（）

**Reply by TKanX** [reply]
好奇作者是不是学生

**Reply by zkz098** [reply]
如果不是的话我大抵是不能"At school"的

**Reply by TKanX** [reply]
我也是，可以一起搞，我会node，express和前端的一些东西。。。

**Comment by D-Sketon** [comment]
furigana和mermaid之外都没用过

**Comment by zkz098** [comment]
目前分包：
基础包：原有文字特效(例如abbr、上标下标等)+excerpt+container+prismjs
spoiler、furigana、graphviz、KaTeX和mermaid独立成包
chart暂时移除，等待社区反馈
